resource_customizations/argoproj.io/Rollout/actions/pause/action.lua

```diff
@@ -0,0 +1,2 @@
+obj.spec.paused = true
```

Comment:
Is this something picked up by the Rollouts controller?

Reply:
Yes, it's picked up by the controller. 
Link to the CRD: https://github.com/argoproj/argo-rollouts/blob/master/manifests/crds/rollout-crd.yaml#L66